docs/installation.rst

```diff
@@ -43,7 +22,7 @@ You can install the latest stable release build of perses via the ``conda`` pack
 
 .. code-block:: none
 
-   $ conda config --add channels omnia
+   $ conda config --add channels conda-forge openeye
```

Comment:
I think we need to still explicitly advise the user install `openeye-toolkits` if we don't currently do so.

Reply:
You are correct!